cluster-autoscaler/proposals/min_at_zero_gcp.md

```diff
@@ -0,0 +1,176 @@
+# Cluster Autoscaler - min at 0 
+### Design Document for Google Cloud Platform
+##### Author: mwielgus
+
+# Introduction
+
+One of the common requests for Cluster Autoscaler (for example:  [1], [2])  is the ability to scale some node groups to zero. This would definitely be a very useful feature but the implementation is somehow problematic in ScaleUP due to couple reasons:
+
+* [P1] There is no live example of what a new node would look like if the currently zero-sized node group was expanded. The node shape is defined as:
```

Comment:
Can we use some other letter for numbering problems (maybe I for 'issue')? I think most people interpret P1 as priority.